Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+                },
+                {
+                    name: "Grisha Lyukshin",
+                    mailto: "glyuk@microsoft.com",
+                    company: "Microsoft"
+                }
+            ],
+            github: "https://github.com/w3c/editing",
+            shortName: "contenteditable-interface",
+            wg: "Web Applications Working Group",
+            wgURI: "https://www.w3.org/2019/webapps/",
+            wgPatentURI: "https://www.w3.org/2004/01/pp-impl/114929/status",
+            license: "w3c-software-doc"
+        };
+    </script>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c-common' class='remove'></script>
+</head>
+
+<body>
+    <section id='abstract'>
+        <h2>Abstract</h2>
+        <p>
+            This specification defines an API that indicates to user agents which editing-related commands are applicable to contenteditable regions of a document.  User agents SHOULD take this information into account when presenting any editing-related UI to ensure its applicability.
+        </p>
+        <p>
+            This specification purposefully does not prescribe the means by which a user agent will indicate to the user that some commands are not applicable in a given contenteditable region.
+        </p>
+    </section>
+    <section id='sotd'>
+      <p></p>
+    </section>
+    <section id='conformance'>
+      <p></p>
+    </section>
+    <section id="introduction">
+        <h2>Introduction</h2>
+        <p>
+            In contenteditable regions of documents, some user agents provide contextual user interface (UI) elements to help accelerate common editing operations.  The UI is meant to enhance the editing experience, but when authors create customized editors that aren’t compatible with user agent UI, it leads to confused users.
+        </p>
+        <p>
+            WYSIWYG markdown editors provide a canonical example. While markdown editors typically support bold or italicized text, they often don't support font color or font family customization. System or browser UI elements like those shown for iOS or the Mac Touch Bar below don't have any indication as to which formatting commands are applicable to a particular contenteditable element and should be displayed, and which ones should be disabled or omitted.
+        </p>
+        <figure>
+            <img src="images/mac-touchbar.png" alt="Mac Touch Bar showing editing buttons">
+            <figcaption>
+                Mac Touch Bar showing editing buttons
+            </figcaption>
+        </figure>
+        <figure>
+            <img src="images/ios-editing-buttons.png" width="300" alt="iOS selection menu showing editing buttons">
+            <figcaption>
+                iOS selection menu showing editing buttons
+            </figcaption>
+        </figure>
+        <p>
+            To address the issue, a new contenteditable-disabled attribute can be applied as shown in the example below.
```

Comment:
🚨 Whoa!!!! Noob question: is this web compatible??? I though "`-whatever`" was reserved for custom elements? Won't this conflict or cause confusion?

Reply:
Also, isn't this a losing proposition? A content author would need to forever keep updating this list of things to disable, while the UA is constantly trying to enable things. 

Wouldn't an all or nothing approach be better?

Reply:
@marcoscaceres You mean a way to turn all off and then turn individual ones on again? We have gone back and forth between blacklisting and whitelisting on this. Browser makers generally prefer blacklisting and content authors generally prefer whitelisting. Browser makers are on the long end of the stick, so we went with blacklisting but with the addition that one can query which items are available, so that with a few lines of JavaScript one should effectively be able to implement whitelisting anyway. 

This querying mechanism seems to have been removed with this PR and there is a tuicket here about bringing it back: https://github.com/w3c/editing/issues/252